### ISSUE TITLE
tar: Ignoring malformed pax extended attribute

### ISSUE THREAD

**Issue by ryandesign** [thestk/stk#140]
stk-5.0.0.tar.gz [does not extract](https://build.macports.org/builders/ports-10.13_x86_64-builder/builds/200502/steps/install-port/logs/stdio
) using the version of BSD tar included with macOS 10.13 and possibly other versions. The error, repeated 552 times, is:

```
tar: Ignoring malformed pax extended attribute
```

followed by:

```
tar: Error exit delayed from previous errors.
```

I just a few hours ago wrote a lengthy analysis of a similar problem in another project:

https://github.com/libwww-perl/Mozilla-CA/issues/21#issuecomment-1668548875

In stk, as was the case for Mozilla::CA, the problematic 5.0.0 archive contains `SCHILY.xattr` attributes whereas the working 4.6.2 archive does not.

**Comment by garyscavone** [comment]
What do you suggest as a "fix"? The problem mainly seems to be with older versions of tar. I generate the tarball with something like:

COPYFILE_DISABLE=true tar -c -vzf stk-x.x.x.tar.gz stk-x.x.x

**Comment by ryandesign** [comment]
As was the case with Mozilla::CA, I believe the problem here for stk is that some of the files you included in the archive had extended filesystem attributes, and your tar program attempted to preserved those attributes in the tar archive using a non-portable method. It seems likely to me that those extended filesystem attributes are not an integral part of your source code distribution, and so I think a fix would be to exclude those extended filesystem attributes when you make your tarballs. I'm not sure which implementation of tar you use but its manpage should explain how that can be accomplished.

**Comment by garyscavone** [comment]
This is fixed in the 5.0.1 release distribution at CCRMA.
